Scripts/SetIfEmpty/SetIfEmpty.py

```diff
@@ -0,0 +1,20 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+
+def main(args):
+    demisto.results(getValueToSet(args))
+
+def getValueToSet(args):
+    value = args.get('value')
+    applyIfEmpty = True if args.get('applyIfEmpty') == 'true' else False
```

Comment:
lets support a case when user write `True`/`TRUE` as well

```suggestion
    applyIfEmpty = True if args.get('applyIfEmpty', '').lower() == 'true' else False
```